app/assets/javascripts/admin/routes/admin-customize-email-style-edit.js.es6

```diff
@@ -0,0 +1,39 @@
+export default Ember.Route.extend({
+  model(params) {
+    return {
+      model: this.modelFor("adminCustomizeEmailStyle"),
+      fieldName: params.field_name
+    };
+  },
+
+  setupController(controller, model) {
+    controller.setProperties({
+      fieldName: model.fieldName,
+      model: model.model
+    });
+    this.set("shouldAlertUnsavedChanges", true);
```

Comment:
This variable is only used in the route and never observed so I'd recommend making it a `this._shouldAlertUnsavedChanges = true` - no need for get/set and it's obvious it's private.